recipes/dunovo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+make
+mkdir $PREFIX/include/dunovo
+cp *.py *.so $PREFIX/include/dunovo
```

Comment:
it doesn't have to be include then ? if can you just change the directory to `$PREFIX/lib/denuvo/` and make the links to bin/ that will be fine.